modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/test/java/com/liferay/portal/security/sso/openid/connect/internal/servlet/filter/backchannel/logout/OpenIdConnectBackchannelLogoutFilterTest.java

```diff
@@ -0,0 +1,301 @@
+				Mockito.mockConstruction(
+					LogoutTokenValidator.class,
+					(mock, context) -> Mockito.when(
+						mock.validate(Mockito.any(JWT.class))
+					).thenReturn(
+						dummyLogoutClaims
+					))) {
+
+			_openIdConnectBackchannelLogoutFilter.processFilter(
+				httpServletRequest, httpServletResponse,
+				Mockito.mock(FilterChain.class));
+
+			LogoutTokenValidator mockValidator = mockedConstruction.constructed(
+			).get(
+				0
+			);
+
+			Mockito.verify(
+				mockValidator
+			).validate(
+				Mockito.any(JWT.class)
+			);
+		}
+
+		Mockito.verify(
+			_openIdConnectSessionLocalService
+		).deleteOpenIdConnectSession(
+			Mockito.eq(openIdConnectSession)
+		);
+
+		Mockito.verify(
+			httpServletResponse
+		).setStatus(
+			HttpServletResponse.SC_OK
+		);
+	}
+
+	private SignedJWT _createIdToken() throws Exception {
+		RSAKey testRSAKey = new RSAKeyGenerator(
+			2048
+		).keyID(
+			_KEY_ID
+		).generate();
+
+		JWSSigner signer = new RSASSASigner(testRSAKey.toPrivateKey());
+
+		Date now = new Date();
+
+		JWTClaimsSet idTokenClaims = new JWTClaimsSet.Builder(
+		).issuer(
+			_ISSUER_URL
+		).audience(
+			_CLIENT_ID
+		).subject(
+			_SUBJECT
+		).issueTime(
+			now
+		).expirationTime(
+			new Date(now.getTime() + 60_000)
+		).claim(
+			"sid", _SESSION_ID
+		).claim(
+			"typ", "ID"
+		).build();
+
+		SignedJWT idToken = new SignedJWT(
+			new JWSHeader.Builder(
+				_JWS_ALGORITHM
+			).keyID(
+				_KEY_ID
+			).build(),
+			idTokenClaims);
+
+		idToken.sign(signer);
+
+		return idToken;
+	}
+
+	private SignedJWT _createLogoutToken() throws Exception {
+		JSONObject eventsClaimValueJSONObject = new JSONObject();
+
+		try {
+			eventsClaimValueJSONObject.put(
+				"http://schemas.openid.net/event/backchannel-logout",
+				new JSONObject());
+		}
+		catch (JSONException jsonException) {
+			throw new RuntimeException(jsonException);
+		}
+
+		RSAKey testRSAKey = new RSAKeyGenerator(
+			2048
+		).keyID(
+			_KEY_ID
+		).generate();
+
+		JWSSigner signer = new RSASSASigner(testRSAKey.toPrivateKey());
+
+		Date now = new Date();
+
+		JWTClaimsSet logoutClaims = new JWTClaimsSet.Builder(
+		).issuer(
+			_ISSUER_URL
+		).audience(
+			_CLIENT_ID
+		).subject(
+			_SUBJECT
+		).issueTime(
+			now
+		).expirationTime(
+			new Date(now.getTime() + 60_000)
+		).jwtID(
+			UUID.randomUUID(
+			).toString()
+		).claim(
+			"sid", _SESSION_ID
+		).claim(
+			"typ", "Logout"
+		).claim(
+			"events", eventsClaimValueJSONObject.toString()
+		).build();
+
+		SignedJWT logoutToken = new SignedJWT(
+			new JWSHeader.Builder(
+				_JWS_ALGORITHM
+			).keyID(
+				_KEY_ID
+			).build(),
+			logoutClaims);
+
+		logoutToken.sign(signer);
+
+		return logoutToken;
+	}
+
+	private HttpServletRequest _mockHttpServletRequest(
+		String logoutTokenSerialized) {
+
+		HttpServletRequest httpServletRequest = Mockito.mock(
+			HttpServletRequest.class);
+
+		Mockito.when(
+			ParamUtil.get(httpServletRequest, "logout_token", StringPool.BLANK)
```

Comment:
The main thing you are mocking here, is the `ParamUtil.get`  method call. The `HttpServletRequest` is just a piece of the whole thing.